mesonbuild/dependencies/ui.py

```diff
@@ -235,18 +237,37 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
             self.compile_args.append('-I' + inc_path)
             self.link_args.append('-L' + lib_path)
             self.link_args.append('-l' + lib_name)
-
-            # TODO: find a way to retrieve the version from the sdk?
-            # Usually it is a part of the path to it (but does not have to be)
-            return
         else:
             # simply try to guess it, usually works on linux
             libs = self.clib_compiler.find_library('vulkan', environment, [])
             if libs is not None and self.clib_compiler.has_header('vulkan/vulkan.h', '', environment, disable_cache=True)[0]:
                 self.is_found = True
                 for lib in libs:
                     self.link_args.append(lib)
-                return
+
+        if self.found:
```

Comment:
The pylint error is telling you that `self.found` is a method object (defined in dependencies/base.py) and you probably meant `self.is_found`.

Reply:
Right, thanks :)